src/main/java/chess/Horse.java

```diff
@@ -10,21 +9,8 @@
         super.setPosition(source);
     }
 
-    public void kill(final Piece target) {
-    }
-
-    public void move(final Piece target) {
-    }
-
-    public void die() {
-    }
-
-    public boolean isPosibleMove(final Piece target) {
-        return true;
-    }
-
-    public List<Piece> getPosibleMoves(final ChessBoard chb) {
-        List<Piece> array = new ArrayList<Piece>();
+    public ArrayList<Position> getPossibleMoves() {
+        ArrayList<Position> array = new ArrayList<Position>();
         return array;
```

Comment:
please change the name array for one more descriptive

Reply:
i made the changes could you review it please?